### PR TITLE
Update docs marked for review

### PR DESCRIPTION
**Pull request by sarahseewhy**
### What

Reviewed and made minor changes to the documentation marked for review.

Also removed a broken command line script and a reference to Concourse which we're no longer actively using.

### Why

Our documentation should not become stale or confusing.

### CODE REVIEW

**Comment by sarahseewhy** [comment]
Apologies, I was committing via the IDE which isn't set up to sign commits hence the "Unverified" marker. I'll set this up next time to rectify the issue.

**Comment by sarahseewhy** [comment]
Figured out how to GPG sign my commits so retroactively signed all of them using `rebase` and they are now "Verified".

File: source/infrastructure/add-new-aws-users.html.md.erb [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+last_reviewed_on: "2022-11-28"
 review_in: 6 months
 ---
 
 # Add new AWS users
 
 You can add new AWS users to the GovWifi AWS accounts by:
 
-1. First verify with the GovWifi developers whether to grant `admin` or `read-only` access.
+1. First verify with the GovWifi developers whether to grant `admin` or `read-only` access. Only team members with SC clearance can be allowed `admin` access. Until SC clearance is passed `read-only` access is allowed.
```
Comment (deanwilson) [inline source/infrastructure/add-new-aws-users.html.md.erb:12]: A tiny nit pick - I _think_ the second clearance is redundant.

File: source/infrastructure/secrets.html.md.erb [hunk 2]
```diff
@@ -98,16 +98,15 @@ Raise a PR in the `govwifi-build` repo on Github. Ask another team member to tes
 
 New members must import all the existing gpg keys that are stored.
 
-To do this read in all the keys and import them:
+To do this read in all the keys in `passwords/.gpg-id` and import them:
 
 ```
-all_key_ids=`cat passwords/.gpg-id`
-for key_id in $all_key_ids; do gpg --keyserver keyserver.ubuntu.com --receive-keys "$key_id"; echo "${key_id}:6" | gpg --import-ownertrust; done
+$ gpg --keyserver keyserver.ubuntu.com --receive-keys <key_id> && gpg --import-ownertrust
```
Comment (deanwilson) [inline source/infrastructure/secrets.html.md.erb:104]: Does having the loop in the doc make it easier to copy and paste for the reader?

**Review by deanwilson — APPROVED** [review]
A couple of minor points but nothing that's blocking a merge.

**Review by szd55gds — APPROVED** [review]
LGTM

File: source/infrastructure/secrets.html.md.erb (hunk 2 above)
Comment (sarahseewhy) [reply source/infrastructure/secrets.html.md.erb:104]: So the loop doesn't work, I tried fixing it but it became a rabbit hole. For now, I think it's sensible to remove the broken loop and then try to improve it when it's next time to review the doc.

I discovered it was broken by doing exactly that: copying and pasting into the terminal and then spending about 20min trying to figure out what was going on.

File: source/infrastructure/add-new-aws-users.html.md.erb (hunk 1 above)
Comment (sarahseewhy) [reply source/infrastructure/add-new-aws-users.html.md.erb:12]: Good spot! I'll make a quick commit to fix that.

**Pull request merged**